examples/list_transactions.rs

```diff
@@ -0,0 +1,29 @@
+use chrono::{Duration, Utc};
+use clap::Clap;
+use monzo::Client;
+
+#[derive(Clap)]
+struct Args {
+    access_token: String,
+}
+
+#[tokio::main]
+async fn main() -> monzo::Result<()> {
+    let args = Args::parse();
+    let client = Client::new(args.access_token);
+
+    let accounts = client.accounts().await?;
+    let account_id = &accounts[0].id;
+
+    let transactions = client
+        .transactions(account_id)
+        .since(Utc::now() - Duration::days(89))
+        .limit(2)
```

Comment:
parameters like `limit` could be exposed as args. that's a small thing though.

Reply:
I've added flags for both the limit and the duration.